tasks/managed/get-push-rpm-params/tests/test-get-push-rpm-params.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-get-push-rpm-params
+spec:
+  description: |
+    Test the get-push-rpm-params task.
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:latest
+            script: |
+              set -eux
+
+              if [ ! -f "$(workspaces.data.path)/data" ] ; then
```

Comment:
this is a brand new pod. You should be able to know with certainty whether or not the dir will already exist (it won't)